it/databases/mssql/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM microsoft/mssql-server-linux
+
+ENV ACCEPT_EULA Y
+ENV SA_PASSWORD sonarsource!2017
```

Comment:
Oups, there is no problem there it's just a default password (and a strong one required by mssql).
Sorry it's not ready for review I've made a mistake.